src/components/molecules/Settings/Section/index.tsx

```diff
@@ -11,51 +11,37 @@ export type Props = {
 const Section: React.FC<Props> = ({ title, actions, children }) => {
   const theme = useTheme();
   return (
-    <Wrapper>
+    <>
```

Comment:
Why can't you use a div here? I am not 100% but to me this is kinda bad for HTML semantics. I know divs don't have meaning(compared to other HTML elements like header, footer, etc) but I don't see a reason not to use a div to clump this related stuff together.
I try to only use React fragments when I absolutely have to or it makes sense (ie. I have a Modal component that will render conditionally outside of the main content of that component).